.github/workflows/scripts/pr-get-job-link.js

```diff
@@ -1,9 +1,13 @@
 
 module.exports = async ({ github, context, core }) => {
+    const name = "Detect breaking changes";
     const { owner, repo } = context.repo;
     const url = `https://api.github.com/repos/${owner}/${repo}/actions/runs/${context.runId}/jobs`
-    const result = await github.request(url)
-    const link = `https://github.com/grafana/grafana/runs/${result.data.jobs[0].id}?check_suite_focus=true`;
-
+    
+    const result = await github.request(url);
+    const job = result.jobs.find(j => j.name === name);
+    const stepIndex = job.steps.findIndex(s => s.name === name);
+    const link = `${job.html_url}#step:${stepIndex + 1}:1`;
```

Comment:
I think this file then has to change too?

- https://github.com/grafana/grafana/blob/main/.github/workflows/detect-breaking-changes-build.yml#L150
- https://github.com/grafana/grafana/blob/main/.github/workflows/detect-breaking-changes-build.yml#L110-L111

Reply:
Ah, thanks! Totally missed that the step got added there.